README.md

```diff
@@ -1,8 +1,151 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams — including leading Layer 2s and rollups — to launch in minutes instead of months.
+Through our API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation across every layer of the ZK app deployment stack.
+
+This repository hosts the images and Helm charts designed to perform proof generation for a Scroll SDK chain.
+The following diagram depicts how the Scroll SDK components work together when you use Sindri as a prover.
+```mermaid
+---
+title: Scroll SDK with Sindri Provers
+---
+flowchart LR
+    subgraph settle[Settlement Layer]
+        direction LR
+    end
+    subgraph sequence[Sequencing Layer]
+        direction LR
+    end
+    subgraph prove[Proving Layer]
+        direction LR
+        C["**Coordinator**"] -.->|Tasks| P@{ shape: processes, label: "**Provers**\nThese components outsource proof generation to Sindri's GPU workers" }
+        P -.->|Proofs| C
+    end
+    subgraph sindri["Scroll Circuits on Sindri"]
+        direction TB
+        Chunk["**Chunk**
+        Verified execution of contiguous blocks (i.e. a zkEVM proof)"]
+        Batch["**Batch**
+        Intermediate aggregation of multiple chunks"]
+        Bundle["**Bundle**
+        Final aggregate proof sent to the settlement layer"]
+    end
+    settle o--o sequence
+    sequence o--o prove
+    P --> Chunk
+    P --> Batch
+    P --> Bundle
+```
```

Comment:
Interesting idea for a dynamically created image, but the end result (as shown) can be confusing/frustrating in terms of not seeing all the text.
<img width="1641" alt="Screenshot 2024-10-28 at 1 51 41 PM" src="https://github.com/user-attachments/assets/44f1d6cb-0fed-4018-9813-04b77c092578">

Reply:
This is interesting. I cannot replicate this in firefox.  No matter what my window size is; or how fast I resize; the text in each box is shown.  The font-size is dynamic for me.  Are you reviewing the README in chrome or some other source?

Regardless, I'd like to keep the graphic. Even if I've got to compromise a bit on the explanatory paragraphs.  The point here is that while your client-side deployment has these pods called "Provers", they are really just relayers between the Coordinator and Sindri workers.

How about this:
1. I revise the text of the "Provers" node to say: "Relays tasks to Sindri workers"
2. I revise the three circuits to say:
  a. Chunk: "zkEVM Circuits"
  b. Batch: "Intermediate Aggregation"
  c. Bundle: "Final On-chain Proof"

Reply:
I will mark this down as a win for firefox.  Since this is only occurring with Safari, I think I'd like to keep the verbose descriptions of circuits.  I think it adds the first inkling of what provers are doing if you have no exposure to zkEVMs otherwise.

Reply:
WFM